src/kili/use_cases/cloud_storage/__init__.py

```diff
@@ -255,9 +255,10 @@ def update_data_integration(
         status: Optional[DataIntegrationStatus],
     ) -> Dict:
         """Update data integration."""
+        formated_organization_id = organization_id if isinstance(organization_id, str) else ""
```

Comment:
should be :

```python
organization_id=OrganizationId(organization_id) if organization_id else None,
```

not: formated_organization_id = organization_id if isinstance(organization_id, str) else ""

Reply:
Yes but if I do it like this i have this error :
`Expression of type "tuple[OrganizationId | None]" cannot be assigned to declared type "str | None"`

Reply:
you  can type the organization_id field of DataIntegrationData  to a Optional[OrganizationId]

Reply:
Oh yes my bad I already did but the formatter didn’t work properly, thanks